extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -71,6 +71,25 @@ public function getTopContributorsData() {
 
 		$topContributorsDetails = $this->getContributorsDetails( $topContributors );
 
+		$query = wfArrayToCGI( [
+			'redirect' => $this->getTitle()->getCanonicalURL(),
+			'uselang' => $this->getLanguage()->getCode(),
+		] );
+
+		$login = Html::element(
+			'a',
+			[ 'href' => 'https://www.wikia.com/signin?' . $query ],
+			$this->msg( 'communitypage-anon-login' )->plain()
+		);
+
+		$register = Html::element(
+			'a',
+			[ 'href' => 'https://www.wikia.com/register?' . $query ],
+			$this->msg( 'communitypage-anon-register' )->plain()
+		);
+
+		$anonText = $this->msg( 'communitypage-anon-contrib-header', $login, $register )->plain();
```

Comment:
Comment from before still applies. The message should be `->escaped()` and use `rawParams` to pass the link parameters instead.